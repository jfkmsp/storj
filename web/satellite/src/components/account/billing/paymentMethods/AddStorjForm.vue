// Copyright (C) 2020 Storj Labs, Inc.
// See LICENSE for copying information.

<template>
    <div class="add-storj-area">
        <p class="add-storj-area__support-info">To deposit STORJ token and request higher limits, please contact <a target="_blank" rel="noopener noreferrer" href="https://supportdcs.storj.io/hc/en-us/requests/new?ticket_form_id=360000683212">Support</a></p>
    </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

// @vue/component
@Component
export default class AddStorjForm extends Vue {
}
</script>

<style scoped lang="scss">
    p {
        margin: 0;
    }

    .add-storj-area {
        margin: 20px 0;
        font-family: 'font_regular', sans-serif;
        display: flex;
        max-height: 52px;
        justify-content: space-between;
        align-items: center;

        &__selection-container {
            display: flex;
            align-items: center;

            &__label {
                margin-right: 30px;
                max-width: 215px;
            }

            &__form {
                width: 60%;
            }
        }

        &__submit-area {
            display: flex;
            align-items: center;
            min-width: 135px;
        }

        &__support-info {
            font-weight: 600;
            font-size: 14px;
            line-height: 20px;
            color: #000;

            a {
                color: #0149ff;
            }
        }
    }

    .loading-image {
        width: 18px;
        height: 18px;
        margin-right: 5px;
    }
</style>
