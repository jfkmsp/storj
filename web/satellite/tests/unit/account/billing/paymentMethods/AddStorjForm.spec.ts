// Copyright (C) 2020 Storj Labs, Inc.
// See LICENSE for copying information.

import { mount } from '@vue/test-utils';

import AddStorjForm from '@/components/account/billing/paymentMethods/AddStorjForm.vue';

describe('AddStorjForm', () => {
    it('renders correctly', () => {
        const wrapper = mount<AddStorjForm>(AddStorjForm);

        expect(wrapper).toMatchSnapshot();
    });
});
