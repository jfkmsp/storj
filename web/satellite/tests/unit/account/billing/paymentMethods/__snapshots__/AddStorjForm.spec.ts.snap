// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddStorjForm renders correctly 1`] = `
<div class="add-storj-area">
  <p class="add-storj-area__support-info">Please contact <a target="_blank" href="mailto:supportdcs@storj.io">supportdcs@storj.io</a> to deposit STORJ tokens into your account</p>
</div>
`;

exports[`AddStorjForm renders correctly after continue To Coin Payments click 1`] = `
<div class="add-storj-area">
  <p class="add-storj-area__support-info">Please contact <a target="_blank" href="mailto:supportdcs@storj.io">supportdcs@storj.io</a> to deposit STORJ tokens into your account</p>
</div>
`;
